lib/mnesia/src/mnesia_loader.erl

```diff
@@ -744,7 +744,14 @@ do_send_table(Pid, Tab, Storage, RemoteS) ->
 	    Storage ->
 		%% Send first
 		TabSize = mnesia:table_info(Tab, size),
-		KeysPerTransfer = calc_nokeys(Storage, Tab),
+		KeysPerTransfer =
+                case ?catch_val(send_table_batch_size) of
+                    {'EXIT', _} -> mnesia_lib:set(send_table_batch_size, 0);
```

Comment:
`{`EXIT` _}` is returned when ets table does not exist (`badarg`).
`mnesia_lib:set` calls `ets:insert`, which will crash

Reply:
Not exactly, it can also return {'EXIT', _}  due to badarg from `ets:lookup_element/2` (not found)